unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -265,18 +266,31 @@ private async UniTask<int> RequestWearablesAndShowThem(int page, CancellationTok
 
                 try
                 {
-                    int customWearablesCount = wearables.Count;
+                    int ownedWearablesCount = wearables.Count;
+                    int ownedWearablesTotalAmount = totalAmount;
```

Comment:
Can we improve these names?

`wearables.Count` is the current page wearables, right? And `totalAmount` is the total amount of wearables of all pages, right?

Maybe we could use:
`ownedWearablesCount` => `currentPageWearablesCount`

Reply:
`wearables.Count` at this point is the amount of wearables retrieved by `GetOwnedWearables` request, thats why i called it `ownedWearablesCount`. Then, the wearables from the builder/custom wearables assigned through query param (&WITH_COLLECTIONS=...&WITH_ITEMS=...) are appended into the list to fill the current page. I need to make the differentiation so i can make calculations on the pagination

Reply:
But `GetOwnedWearables` (I think you mean `RequestOwnedWearablesAsync`) has a `page` param, so isn't that just the wearables of that page?

Reply:
The wearable count of the current page is: ownedWearables(page) + customWearables. Since customWearables might overload the page size (15) we have to manually calculate the pagination